### PR TITLE
Do not use file pointers in non-atomic way

### PR DESCRIPTION
**Pull request by callegar**

### CODE REVIEW

**Comment by callegar** [comment]
This probably fixes the "random" md5sums in bug #2

File: src/nlinkfs.c [hunk 1]
```diff
@@ -446,8 +446,7 @@ static int nlinkfs_open(const char *path, struct fuse_file_info *fi)
  */
 static int nlinkfs_read(const char *path, char *buf, size_t size, off_t offset, struct fuse_file_info *fi)
 {
-	lseek(fi->fh, offset, SEEK_SET);
-	return read(fi->fh, buf, size);
+        return pread(fi->fh, buf, size, offset);
```
Comment (rene) [inline src/nlinkfs.c]: Please, fix indentation here, there are extra spaces at the begin and one extra space at the end of this line.

**Comment by rene** [comment]
@callegar , I tested your PR, it solves the problem. Please, sign off your commit and address my comment and we are ready for merge.

**Comment by callegar** [comment]
Sorry, I have almost no experience with pull requests, I'll look at what needs to be done to put it in shape!

**Comment by callegar** [comment]
OK now?

**Comment by rene** [comment]
@callegar , thanks for the PR, I just improved the commit message and merged to master.

**Pull request closed without merging**